sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ece-request/ece-process-integration.scheduler.ts

```diff
@@ -0,0 +1,49 @@
+import { InjectQueue, Process, Processor } from "@nestjs/bull";
+import { ECEFileService } from "@sims/integrations/institution-integration/ece-integration";
+import { QueueService } from "@sims/services/queue";
+import { QueueNames } from "@sims/utilities";
+import { InjectLogger, LoggerService } from "@sims/utilities/logger";
+import { Job, Queue } from "bull";
+import { BaseScheduler } from "../../base-scheduler";
+import {
+  GeneratedDateQueueInDTO,
+  ECEProcessResultQueueOutDTO,
+} from "./models/ece.model";
+
+@Processor(QueueNames.ECEProcessIntegration)
+export class ECEProcessIntegrationScheduler extends BaseScheduler<GeneratedDateQueueInDTO> {
+  constructor(
+    @InjectQueue(QueueNames.ECEProcessIntegration)
+    schedulerQueue: Queue<GeneratedDateQueueInDTO>,
+    private readonly eceRequest: ECEFileService,
+    queueService: QueueService,
+  ) {
+    super(schedulerQueue, queueService);
+  }
+
+  /**
+   * Identifies all the applications that have an eligible COE request waiting
+   * for a particular institution and generate the request file.
+   * @params job has generatedDate Date in which the assessment for
+   * particular institution is generated.
+   * @returns Processing result log.
+   */
+  @Process()
+  async processECERequest(
+    job: Job<GeneratedDateQueueInDTO | undefined>,
+  ): Promise<ECEProcessResultQueueOutDTO[]> {
```

Comment:
We are no longer using DTOs for this, please use the `QueueProcessSummaryResult`.

Reply:
And also use summary.info instead of logger.log here.

Reply:
> 

if only it's needed to be in the result. else use `this.logger.log` and `job.log`